config/components/manager/controller_manager_config.yaml

```diff
@@ -16,6 +16,9 @@ controller:
     ClusterQueue.kueue.x-k8s.io: 3
     ResourceFlavor.kueue.x-k8s.io: 3
     Workload.kueue.x-k8s.io: 3
+client:
+  qps: 25
+  burst: 35
```

Comment:
A bit too shy? :)

I think we can match kube-scheduler's default of qps=50, burst=100

https://github.com/kubernetes/kubernetes/blob/4802d7bb62c2623be8e4f940f6b5c1fcddd6c744/pkg/scheduler/apis/config/v1/defaults.go#L145

Reply:
Done.